styles/style.css

```diff
@@ -384,3 +384,77 @@ li {
   font-size: 1.3em;
   margin: 5% 0;
 }
+
+/* Contact Me Form */
+#aboutme {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #e5e5e5;
+  height: 100vh;
+  margin-top: 5%;
+}
+
+.abtme-title {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 2em;
+  width: 10em;
+  text-align: center;
+  margin: 5%;
+  color: #172b4d;
+}
+
+.input-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+input,
+textarea {
+  width: 80%;
+  margin-bottom: 4%;
+  border: none;
+  resize: none;
+  background-color: #e5e5e5;
+  padding: 4%;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 1em;
+}
+
+.input-border {
+  border-bottom: 1px solid #dbd8d7;
+}
+
+.abtme-btn-container {
+  display: flex;
+  justify-content: center;
+}
+
+/* Footer */
+.footer-bar {
+  background-color: #aaacb0;
+  width: 100%;
+  height: 2px;
+  margin: 10% 0% 0% 2%;
+}
+
+footer {
+  background-color: #e5e5e5;
+  margin-top: 10%;
+}
+
+.footer-media {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  height: 1.1em;
+}
+
+.footer-media li {
+  margin-right: 2%;
+}
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So kindly adjust the font color, the background color , the text and the form input alignment and social icons just like the figma design below. 

- The Figma Design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/220424584-e29059d8-bbc2-4960-988c-44b163c2cc55.PNG)

- Your Project Design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/220424881-ce339dfa-15ee-4511-bc7d-26c15b545db7.PNG)

Reply:
@Abenezer-Tilahun Hey! I´ve implemented the changes and now I´m waiting for a new review, I wanted to know if there's something you can help me with, I´ve been waiting for 3 hours now